WISE-Community/WISE-Client
# feat(Run): Allow teachers to archive runs

## Changes

- Teachers can now archive runs
- Removed filter by period select drop down in Class Schedule view
- There is now an active runs view and an archived runs view

## Test

- Test with https://github.com/WISE-Community/WISE-API/pull/225

- Archive/unarchive a run using the run item menu list (the 3 dots menu on a run)
- Archive/unarchive a run using the run item checkboxes and then the "Archive Selected"/"Unarchive Selected" button
- Archive/unarchive a run that was shared with you
- Make sure the select all runs checkbox works properly
- Make sure the select runs drop down works properly for (all, none, running, completed)
- Make sure switching from the active to archived view works properly

Closes #1012
